.github/workflows/_test.yml

```diff
@@ -5,24 +5,54 @@ on:
 permissions: {}
 
 jobs:
+  lint-test:
+    name: Lint Test
+    strategy:
+      matrix:
+        python-version: [3.x]
+
+    env:
+      TOXENV: lint
```

Comment:
```suggestion
```
No need for this. You only run one toxenv on one python-version. You can specify them directly below.

Reply:
That's right. Updated to follow your thoughts.